pwiz_tools/Skyline/TestTutorial/CustomReportsTutorialTest.cs

```diff
@@ -482,7 +484,28 @@ protected bool DoQualityControlSummaryReports()
             PauseForScreenShot<ViewEditor.FilterView>("Customize View - Filter tab", 23);
             OkDialog(viewEditor, viewEditor.OkDialog);
 
-            RunUI(() => documentGridForm.DataGridView.Columns[6].Width = 105);
+            RunUI(() =>

```

Comment:
Note that something like this can also go inside a RunUIForScreenShot() call to keep from having to do such minor irrelevant tweaks during normal test runs.

Reply:
Updated. And used in a few other places. Thx for the pointer.